hack/dockerfiles/test.Dockerfile

```diff
@@ -154,7 +155,8 @@ FROM buildkit-buildkitd AS rootless
 RUN apk add --no-cache shadow shadow-uidmap \
   && useradd --create-home --home-dir /home/user --uid 1000 user \
   && mkdir -p /run/user/1000 /home/user/.local/tmp /home/user/.local/share/buildkit \
-  && chown -R user /run/user/1000 /home/user
+  && chown -R user /run/user/1000 /home/user \
+  && rm /bin/su && ln -s /bin/busybox /bin/su
```

Comment:
Probably, we can just lock the root with `passwd -l root`: https://github.com/genuinetools/img/blob/151b4506829bd7f52f5ba83aa99a3fd08709554a/Dockerfile#L53